style.css

```diff
@@ -355,3 +356,151 @@ body {
 .arrow1 {
   margin-left: 205px;
 }
+
+.contact {
+  padding-top: 80px;
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #6070ff;
+  background-image: url(./img/contact\ form\ shapes-mobile.svg);
+  background-repeat: no-repeat;
+  background-position: 130% 20%;
+  border-radius: 75px 0 0 0;
+}
```

Comment:
- [ ] You did a great job implementing the changes :wink: However as mentioned by the previous reviewer, there is some extra gap (gray color space) at the left-hand side of your contact section. Kindly remove that gap to match your design with Figma's design.

_**HIT:** Add `margin-left: -25px;` to this `.contact` class to remove the gap at the left side of your contact section._

### Your Design
![230162266-f056c665-dd29-4d7f-86d4-4eba19e735d6](https://user-images.githubusercontent.com/91142432/230212423-bd4d4b4a-2480-484f-a193-ab3637e96e6e.png)

### Figma's Design
![230162340-bbaf6920-5e5c-47df-a546-46f1a9b6414f](https://user-images.githubusercontent.com/91142432/230212496-4e0292a8-49dd-4060-a998-23a06c3d846b.png)